crates/sui-faucet/src/main.rs

```diff
@@ -50,7 +50,7 @@ struct FaucetConfig {
     #[clap(long, default_value_t = 10)]
     request_buffer_size: usize,
 
-    #[clap(long, default_value_t = 120)]
+    #[clap(long, default_value_t = 30)]
```

Comment:
30s is probably still too long. Normal users (who request coins from wallet UI) wouldn't wait for 30 seconds before getting frastrated

Reply:
My thinking here is that timeout should be a last line of defence, and we should have alerting in place at, e.g. the 10s mark so we know when service has degraded to a point that is irritating for people using wallet.  Then having a window between 10s and 30s gives us a chance to weather hot spots in our load without doing anything (apart from knowing about it).

i.e. I don't want to set the timeout to the maximum acceptable time (which I think falls in the 5-10s range).  Thoughts?